tests/Feature/WebServiceSoapServiceCallerTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Tests\Feature\Api;
+
+use Illuminate\Foundation\Testing\WithFaker;
+use Mockery;
+use ProcessMaker\Contracts\SoapClientInterface;
+use ProcessMaker\Managers\WebServiceSoapServiceCaller;
+use Tests\TestCase;
+
+class WebServiceSoapServiceCallerTest extends TestCase
+{
+    use WithFaker;
+
+    /**
+     *
+     * @var WebServiceSoapServiceCaller
+     */
+    private $manager;
+
+    protected function setUp(): void
+    {
+        parent::__construct();
```

Comment:
@caleeli This should be `parent::setUp()`

Or even better, use you can use `setUpXXX` like

```
    protected function setUpManager()
    {
        $this->manager = new WebServiceSoapServiceCaller;
    }
```

Reply:
ohh! thanks! fixed